mtools/mlaunch/mlaunch.py

```diff
@@ -278,13 +278,14 @@ def init(self):
 
         # Check if config replicaset is applicable to this version
         current_version = self.getMongoDVersion()
-        if self.args['csrs']:
-            if LooseVersion(current_version) < LooseVersion("3.2.0"):
-                errmsg = " \n * The '--csrs' option requires MongoDB version 3.2.0 or greater, the current version is %s.\n" % current_version
-                raise SystemExit(errmsg)
 
+        # Exit with error if --csrs is set and MongoDB < 3.2.0
+        if self.args['csrs'] and LooseVersion(current_version) < LooseVersion("3.2.0"):
```

Comment:
Minor, but should the loose version check actually be 3.1.0 since CSRS was added in the 3.1 dev series? I think it's fine for the error messages to refer to production versions.